hub-templates/daskhub/Chart.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
-- name: base-hub
-  repository: file://../base-hub
-  version: 0.0.1
+- name: basehub
+  repository: file://../basehub
+  version: 0.0.1-n569.hb296398
```

Comment:
I was a bit confused about the basehub version here.
The `.lock` file is generated based on the `.yaml` file right? Because in `daskhub/Chart.yaml` the basehub dependency is set to just `0.0.1`.

Also, when reading about this `Chart.lock` file, I learned that these are the helm3 equivalents of `requirements.lock` and `.yaml`. But we have both for our helm charts. Is there any reason for this?

Reply:
I think just removing them is probably the way to go actually.